problem_builder/public/js/answer.js

```diff
@@ -35,9 +35,11 @@ function AnswerBlock(runtime, element) {
             if (result.status) {
                 if (result.status === "correct") {
                     checkmark.addClass('checkmark-correct icon-ok fa-check');
+                    checkmark.attr('aria-label', checkmark.data('label_correct'))
```

Comment:
Nit: missing semicolon at end of statement. Not a big deal, but I think edX JS styleguide says to always use semicolons.
Semicolons are missing in some other lines of JS code that you added.